orcestra_book/ctd.md

```diff
@@ -0,0 +1,42 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# CTD measurements
+
+[CTD](https://www.awi.de/en/science/biosciences/deep-sea-ecology-and-technology/technology/water-sampler.html) (conductivity, temperature, and depth) measurements are an essential tool used oceanography.
+
+During BOWTIE, 82 CTD measurements were carried out at different locations in the tropical Atlantic:
+
+```{code-cell} ipython3
+import cartopy.crs as ccrs
+import cmocean
+import matplotlib.pyplot as plt
+import xarray as xr
+
+
+ds = xr.open_dataset("ipns://latest.orcestra-campaign.org/products/METEOR/CTD.zarr", engine="zarr")
```

Comment:
I'd recommend to access data using CIDs instead of the IPNS name (e.g. as in https://orcestra-campaign.org/iwv_comparison.html) to make the plot reproducible. This

* helps to educate people about the existence of CIDs
* ensures that the notebook doesn't accidentally change at a random point in time
* ensures that the notebook is rebuilt once we actively decide to update the data shown